packages/@tailwindcss-upgrade/src/codemods/config/migrate-js-config.ts

```diff
@@ -406,15 +440,11 @@ const ALLOWED_THEME_KEYS = [
   // Used by @tailwindcss/container-queries
   'containers',
 ]
-const BLOCKED_THEME_KEYS = ['supports']
 function onlyAllowedThemeValues(theme: ThemeConfig): boolean {
   for (let key of Object.keys(theme)) {
     if (!ALLOWED_THEME_KEYS.includes(key)) {
       return false
     }
-    if (BLOCKED_THEME_KEYS.includes(key)) {
-      return false
-    }
```

Comment:
👋 bye bye blocked theme keys

We still have `aria`, `data` and `supports` handling in: https://github.com/tailwindlabs/tailwindcss/blob/feat%2Fupgrade-supports-theme/packages/tailwindcss/src/compat/theme-variants.ts

This is still necessary in case the config file could not be migrated due to other issues. So didn't get rid of that part.